akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1027,7 +1029,7 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
             "Leader can currently not perform its duties, reachability status: [{}], member status: [{}]",
             membershipState.dcReachabilityExcludingDownedObservers,
             latestGossip.members.collect {
-              case m if m.dataCenter == selfDc ⇒
+              case m if m.dataCenter == cluster.selfDataCenter ⇒
```

Comment:
this is used in so many places and rather verbose, I'd prefer `def selfDc = cluster.selfDataCenter`